packages/bot-web-ui/src/stores/app-store.js

```diff
@@ -1,16 +1,29 @@
-import { action, reaction } from 'mobx';
+import { action, reaction, makeObservable } from 'mobx';
 import { isEuResidenceWithOnlyVRTC, showDigitalOptionsUnavailableError } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 import { runIrreversibleEvents, ApiHelpers, DBot } from '@deriv/bot-skeleton';
 
 export default class AppStore {
     constructor(root_store) {
+        makeObservable(this, {
+            onMount: action.bound,
+            onUnmount: action.bound,
+            onBeforeUnload: action.bound,
+            registerReloadOnLanguageChange: action.bound,
+            registerCurrencyReaction: action.bound,
+            registerOnAccountSwitch: action.bound,
+            registerLandingCompanyChangeReaction: action.bound,
+            registerResidenceChangeReaction: action.bound,
+            setDBotEngineStores: action.bound,
+            onClickOutsideBlockly: action.bound,
+            showDigitalOptionsMaltainvestError: action.bound,
+        });
+
         this.root_store = root_store;
```

Comment:
Please log `this.root_store` to make sure that it isn't undefined!

Reply:
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/97010868/200807844-384625b5-9996-4541-a624-079278a1d2d5.png">

Reply:
i can see the values coming on console.log for the store or do you mean to put an undefined check ?

Reply:
base on your log it is working properly, thank u for checking that

Reply:
I had an issue during migration and all of my root_stores were undefined so I had many errors in the project
I just noticed that to make sure that you don't have the same issue